CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * [ObjectServer] Added `SyncSession.uploadAllLocalChanges()`.
 * Added `Nullable` annotation to methods that may return `null` in order to improve Kotlin usability. This also introduced a dependency to `com.google.code.findbugs:jsr305`.
+* Added support for new data type `MutableRealmIntegers`.  The new type functions almost exactly as a reference to a Long (mutable nullable, etc) but supports `increment` and `decrement` methods, which implement a Conflict Free Replicated Data Type, whose value will converge even when changed across distributed devices with poor connections.
```

Comment:
There is two spaces after `Added support for new data type MutableRealmIntegers.`